t/14-isotovideo.t

```diff
@@ -285,4 +294,5 @@ done_testing();
 END {
     rmtree "$Bin/data/tests/product";
     unlink("$data_dir/wheels.yaml") if -e "$data_dir/wheels.yaml";
+    rmtree "$pool_dir/writer";
```

Comment:
This is not necessary. The `$pool_dir` is a subdirectory of the tempdir which gets removed automatically at the end.

Reply:
I think it is actually generally a good idea to do cleanup before running tests because you can never rely on the cleanup that happens at the end of the test (e.g. the code you're testing is indeed broken and therefore the test crashes in the middle).

Reply:
`tempdir()` creates a directory with random characters, and I don't see why we should delete a directory in the `END` block if the top directory is removed automatically anyway. And I don\t understand how your comment about cleanup up at the beginning of the test relates to that.

We now have a useless `rmtree` there.

Reply:
resolved by https://github.com/os-autoinst/os-autoinst/pull/2138